docs/lib/datastore/fragments/flutter/relational/updated-schema.md

```diff
@@ -0,0 +1,26 @@
+For the examples below with DataStore let's add a new model to the [sample schema](~/lib/datastore/getting-started.md#sample-schema):
```

Comment:
Is this schema actually different across the platforms? Should we be forking it via `./docs/lib/datastore/fragments` or can we re-use a common schema across at least Android, iOS, Flutter?

Reply:
The underlying graphQL schema syntax should be the same so I think it would make sense to re use a common schema.